logic/src/commonTest/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepositoryTest.kt

```diff
@@ -437,6 +438,11 @@ class ConversationRepositoryTest {
             .whenInvokedWith(anything(), anything())
             .thenDoNothing()
 
+        given(conversationDAO)
+            .suspendFunction(conversationDAO::getConversationByQualifiedID)
+            .whenInvokedWith(anything())
+            .then { TestConversation.VIEW_ENTITY }
+
```

Comment:
I might sound like a scratched vinyl, but if you have time, adapting this test to the Arrangement pattern will save us a lot of boilerplate code and make it easier to add new one :)

Reply:
i was afraid you ask for that XD